portal/src/ReactApp.tsx

```diff
@@ -78,11 +82,34 @@ async function loadSystemConfig(): Promise<SystemConfig> {
   return instantiateSystemConfig(mergedConfig);
 }
 
+function isPosthogResetGroupsException(ex: SentryException) {
+  return (
+    ex.type === "TypeError" &&
+    ex.value === "posthog.resetGroups is not a function"
+  );
+}
+
+// DEV-1767: Unknown cause on posthog error, silence for now
+function sentryBeforeSend(event: SentryEvent) {
+  return {
+    ...event,
+    exception:
+      event.exception != null
+        ? {
+            values: event.exception.values?.filter(
+              (value) => !isPosthogResetGroupsException(value)
+            ),
+          }
+        : undefined,
+  };
+}
```

Comment:
From the doc https://docs.sentry.io/platforms/javascript/configuration/filtering/#using-before-send It says return null to discard. Did I misunderstand anything?

Reply:
I was mistaken that one event can have other exceptions. Fixed